configuration/restdocs.md

```diff
@@ -7,14 +7,29 @@ title: openHAB REST API
 
 # openHAB REST API
 
-The REST API of openHAB serves different purposes.
-It can be used to integrate openHAB with other systems as it allows read access to items and item states as well as status updates or the sending of commands for items.
-Furthermore, it gives access to the sitemaps, so that it is the interface to be used by remote user interfaces (e.g. fat clients or javascript based web clients).
-The REST API also supports server-push, so you can subscribe yourself on change notification for certain resources.
+The REST API gives the ability to access a wide range of information associated with a running openHAB installation, and the ability to invoke changes to the openHAB installation. 
+The REST API provides an interface that may only be accessed programmatically.
+The information available from the API includes Items and their State, installed bindings, sitemap information, and much more. 
+The information exchange takes place through HTTP and the API provides a defined protocoll for information exchange among different computer systems. 
+However, from a technical perspective it is very easy to use. 
+Be aware that the REST API is not automatically installed in openHAB. 
+After the REST API documentation add-on is installed, it is accessible through the openHAB dashboard. 
+Access the REST API by connecting any browser to: http://123.456.789.000/8080/doc/index.html (with 123.456.789.000 being the ip address of the computer that runs openHAB). 
+Access over the Internet to the REST API is possible but this represents a significant security risk, and users are encouraged to ensure safe and secure connections. 
+Some ways this can be achieved include using [myopenHAB cloud services](http://docs.openhab.org/installation/security.html#myopenhab-cloud-service), a [reverse proxy server](http://docs.openhab.org/installation/security.html#nginx-reverse-proxy), or a VPN.
```

Comment:
The whole article would look better with a few headlines.

Reply:
As I said before, I don't think the way how a user accesses the REST API belongs here. We would need to add similar content to the Basic UI, Paper UI, Classic UI, Android app, iOS app, ... articles. There is already one consolidated article discussing the whole issue. You can simply link it here 😃  http://docs.openhab.org/installation/security.html

Reply:
+1  re: Headlines, perhaps it is best to get the content down and then add the headlines?

+1 deleting the information about accessing and just pointing to the security article.

Reply:
done, removed the individual examples; added headlines, but it is a short article not sure how many you had in mind